ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -339,7 +360,17 @@ open class BadgeView: UIView {
     open override func layoutSubviews() {
         super.layoutSubviews()
         backgroundView.frame = bounds
-        label.frame = bounds.insetBy(dx: -size.horizontalPadding, dy: -size.verticalPadding)
+
+        if let avatar = dataSource?.avatar {
```

Comment:
Curious: have you validated Right To Left (RTL) languages to ensure the view will be flipped?

Reply:
Updated for RTL